src/components/RouteWrapper.tsx

```diff
@@ -581,6 +582,13 @@ export const ROUTES: RouteProperties[] = [
             "WPScan is an enumeration tool that scans remote WordPress installations in attempt to identify security issues.",
         category: "Web Application Testing",
     },
+    {
```

Comment:
Please place in alphabetical order, so it should be between SnmpCheck and The Harvester.